packages/realm/android/src/main/cpp/lib/arm64-v8a

```diff
@@ -0,0 +1 @@
+../../../../../../realm_dart/binary/android/arm64-v8a/
```

Comment:
Can we have a single symlink in `packages/realm/android/src/main/cpp/lib` pointing `../../../../../../realm_dart/binary/android/`? Or is there something else in the source dir we don't want to link here?

Reply:
No, and we could. I was just doing the least intrusive change. Will fix.

Reply:
Fixed